package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --fix --report-unused-disable-directives --max-warnings 0",
```

Comment:
I'm not sure you actually want to use `--fix` here. You would like `lint` to report an errors. You could create a separate script for fixing named `lint:fix`.

Reply:
I have added a separate script for fixing the lint.